restapi/src/test/java/io/stargate/web/docsapi/service/ReactiveDocumentServiceTest.java

```diff
@@ -197,13 +197,14 @@ public void happyPath() throws Exception {
               eq(collection),
```

Comment:
This class is missing 5 tests updates for the TTL:

* write with ttl
* write batch with ttl
* update with ttl
* update sub-path with ttl auto
* patch with ttl auto